doc/modules/changes/20190526_thieulot

```diff
@@ -0,0 +1,2 @@
+Added polydiapirs benchmark (2019/05/26)
+
```

Comment:
You need to exactly follow the formatting of the other files, otherwise your entry will look broken on the website (this is stuff that goes directly to the following page: https://aspect.geodynamics.org/doc/doxygen/changes_current.html)